tool/net/redbean.c

```diff
@@ -2631,6 +2647,8 @@ void RedBean(int argc, char *argv[]) {
   inbuf.p = xvalloc(inbuf.n);
   hdrbuf.n = 4 * 1024;
   hdrbuf.p = xvalloc(hdrbuf.n);
+  // TODO: Maybe make this an optional argv?
```

Comment:
I'd recommend `-b`. All you have to do is add it to the `getopt()` invocation as well as the man page string near the top of the redbean.c file. We need it because redbean is designed to be automated. I've been working on unit tests that launch it as a subprocess within a unit test, using the `redbean.com -zp0` flags which bind to a random port and then prints it to stdout.

Here's the work in progress on the integration test: https://github.com/jart/cosmopolitan/blob/master/test/tool/net/redbean_test.c Most of the other tests for redbean are stored in https://github.com/jart/cosmopolitan/tree/master/test/net/http right now.

Reply:
Another thing you might consider is adding a Lua API that plugs into your launcher function. That way, if someone wants to distribute a redbean that doesn't need the `-b` flag, they can just put something like `LaunchBrowser()` in their `/tool/net/.init.lua` file which would make launching the browser the default behavior.

Reply:
That's a really good idea, I think having to pass the -b flag is a little cumbersome, as it would be nice to just "double click" the exe (which obviously won't pass any args) but still have a browser launch, so adding the lua init would fix that.

Reply:
Oh right that's true. Judging by your reaction, I think you're instinct is right. The Lua API alone seems perfect. Let's hold off on the flag unless someone explicitly requests it.